.github/workflows/testadapters.yaml

```diff
@@ -66,13 +82,12 @@ jobs:
               echo '{ "meshery-provider": "None", "token": "" }'  > ~/.meshery/auth.json
             fi
             cat ~/.meshery/auth.json
-
         
       # Start meshery server (Assuming that this step will create the meshery namespace)
       - name: Start meshery server using mesheryctl
         run: |
-           curl -L https://git.io/meshery | PLATFORM=kubernetes bash -
-           sleep 20s
+              curl -L https://git.io/meshery | PLATFORM=kubernetes bash -
+              sleep 20s
```

Comment:
No sleeping.

Reply:
I will remove this one. But some of the sleeping is required at least 30 second total